### ISSUE TITLE
Bug Report for Files app

### ISSUE THREAD

**Issue by shawn-fetanat** [files-community/Files#5705]
### Description

I tried to open my Files app and it all of a sudden crashed. Then it gave me a message that said "A problem occurred that developers are not prepared for. Would you like to report the bug." Since that has happened I now can't open Files at all it just won't open now and it only gave me the message about reporting the bug the first time around.
https://www.loom.com/share/d71820b8d6554627a807c1c7edb3c942

### Steps To Reproduce

All I did was click Files to open it. Then it randomly crashed.

### Expected behavior

Expected behavior was for Files to open when I click it not crash.

### Files Version

v1.5.0.0

### Windows Version

Windows 10 21H1

**Comment by shawn-fetanat** [comment]
I tried un-installing and re-installing but now when I go to Microsoft Store and try to download it says "Try again later. Something happened our end."

**Comment by gave92** [comment]
Thanks for this report. Pleas add the debug files to the issue description.

**Comment by shawn-fetanat** [comment]
Thank you.

**Comment by shawn-fetanat** [comment]
I just tried downloading and installing it using the Files package link you provided. That did not work either. When I click install it does something but when look for app after nothing comes up. It's ok maybe I just need to do a restart. I will update you if that does not work either.

**Comment by shawn-fetanat** [comment]
So I don't know why this happened but when Files first crashed, for some reason my Microsoft Store Install Service (in my Services) was set to disabled. That's why I wasn't able to download anything from the Microsoft Store and why it kept crashing and saying "Try again something went wrong on our end." I was able to resolve this issue by going into my Windows Services, then going to "Microsoft Store Install Service," then double clicking that. Then I set my startup type for that service from "disabled" to "manual." Then under service status I clicked to start to restart the service and after that I was able to download apps again from Microsoft Store  (including your app) without it crashing.
